src/Microsoft.Framework.FileSystemGlobbing/FilePatternMatch.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using Microsoft.Framework.Internal;
+
+namespace Microsoft.Framework.FileSystemGlobbing
+{
+    public class FilePatternMatch : IEquatable<FilePatternMatch>
+    {
+        public string Path { get; }
+        public string Stem { get; }
+
+        public FilePatternMatch(string path, string stem)
+        {
+            Path = path;
+            Stem = stem;
+        }
+
+        public bool Equals(FilePatternMatch other)
+        {
+            return other != null &&
+                string.Equals(other.Path, Path, StringComparison.OrdinalIgnoreCase) &&
```

Comment:
In case we're going to switch between case sensitive and insensitive in the future, we better prepare for that.